### DISCUSSION TITLE
does allure produce two videos ?

### DISCUSSION THREAD

**Discussion by AmirAmArA** [Shelex/cypress-allure-plugin#153] [Q&A] [answered]
hey everyone, 
when trying to run my tests i get two videos, in my allure results folder, one's name is hashed and the other is the name of the test with an mp4 format, i can't seem to find where is producing two videos, any ideas ?

**Comment by Shelex** [comment]
Hi @AmirAmArA 
It should be just that one with hashed name 🤔 
Maybe you can provide a repo with example where we can reproduce it and investigate properly?

Or you can debug yourself - [debugging](https://github.com/Shelex/cypress-allure-plugin#debugging), by adding env var before your run command, like `DEBUG=allure-plugin* npx cypress run` to identify which functionality is responsible for attaching odd videos.

**Reply by AmirAmArA** [reply]
using these flags in cypress.config.ts fixed my problem,
what's commented out is what i removed, inside the allure writer it will produce the video 
```
video: true,
  downloadsFolder: 'cypress/fixtures/downloads',
  numTestsKeptInMemory: 1,
  // videosFolder: 'allure-results',
```

**Comment by AmirAmArA** [comment, marked as answer]
using these flags in cypress.config.ts fixed my problem,
what's commented out is what i removed, inside the allure writer it will produce the video

```
video: true,
  downloadsFolder: 'cypress/fixtures/downloads',
  numTestsKeptInMemory: 1,
  // videosFolder: 'allure-results',
```
